backend/dbinit/init.sql

```diff
@@ -0,0 +1,40 @@
+CREATE DATABASE IF NOT EXISTS messageboard;
+
+USE messageboard;
+
+DROP TABLE IF EXISTS Messages;
+
+CREATE TABLE Messages (
+    ip VARCHAR(255) DEFAULT NULL,
```

Comment:
Means the `ip` would take "VINAY" as input also ?? 
Should we implement the validation at Frontend side ?
I want that only `ip` which starts with `172.16.155.x` can access the website. No other input should.